kaggle/api/kaggle_api_extended.py

```diff
@@ -3982,6 +3998,22 @@ def get_or_default(self, data, key, default):
       return data[key]
     return default
 
+  def get_bool(self, data, key, default=False):

```

Comment:
JM2C based on the comment above, I'd kind of suggest forcing the caller to specify the `default` value.  The analogous `get_or_default` makes the caller specify `None`.